### PR TITLE
Update zh_cn.json

### PR DESCRIPTION
**Pull request by OrzMiku**

### CODE REVIEW

File: api/src/main/resources/assets/nolijium/lang/zh_cn.json [hunk 1]
```diff
@@ -83,13 +106,15 @@
...
-    "nolijium.options.disable_particles.tooltip": "防止所有粒子渲染。",
+    "nolijium.options.disable_particles.tooltip": "阻止所有粒子渲染。",
     "nolijium.particle_id": "允许 %s 粒子。",
     "nolijium.chroma": "彩色模式",
     "nolijium.options.chroma_speed.name": "色彩变化速度",
     "nolijium.options.chroma_speed.tooltip": "较高的值循环速度较快，较低的值循环速度较慢。",
     "nolijium.options.enable_chroma_block_outlines.name": "启用彩色方块轮廓",
     "nolijium.options.enable_chroma_block_outlines.tooltip": "启用方块轮廓上的彩色效果。",
+    "nolijium.options.chroma_block_shape_overlay.name": "启用彩色方块形状覆盖",
+    "nolijium.options.chroma_block_shape_overlay.tooltip": "如果设置，将在方块表面绘制彩色覆盖，使用指定值作为透明度。\n在启用§oFabulous!§r渲染时可能会在某些方块上出现异常。",
```
Comment (Nolij) [inline api/src/main/resources/assets/nolijium/lang/zh_cn.json:117]: This is copied from the vanilla quality level translation. Does the vanilla `zh_cn` translation not include this?

File: api/src/main/resources/assets/nolijium/lang/zh_cn.json [hunk 2]
```diff
@@ -41,18 +64,18 @@
     "nolijium.options.hud_shadow.name": "HUD 文本阴影",
     "nolijium.options.hud_shadow.tooltip": "如果启用，HUD 文本将带有阴影。",
     "nolijium.options.hud_refresh_rate_ticks.name": "HUD 刷新率",
-    "nolijium.options.hud_refresh_rate_ticks.tooltip": "HUD 刷新内容的频率。\"充分利用整个 CPU\" 选项将每帧刷新一次。较低的值可能会降低性能。",
+    "nolijium.options.hud_refresh_rate_ticks.tooltip": "HUD 刷新内容的频率。 §l\"充分利用整个 CPU\"§r 选项将每帧刷新一次。较低的值可能会降低性能。",
```
Comment (Nolij) [inline api/src/main/resources/assets/nolijium/lang/zh_cn.json:67]: Were the quotes kept intentionally? I removed them from the `en_us` translation. If they were kept intentionally for readability in the `zh_cn` locale that's fine, just wanted to check.

File: api/src/main/resources/assets/nolijium/lang/zh_cn.json (hunk 1 above)
Comment (OrzMiku) [reply api/src/main/resources/assets/nolijium/lang/zh_cn.json:117]: Oh! It was an oversight on my part.

**Comment by OrzMiku** [comment]
I fixed the issues and it should be fine now.

**Review by Nolij — APPROVED** [review]

**Pull request merged**